plugins/ruby-foreman-tasks/debian/gem.list

```diff
@@ -1,12 +1,12 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman-tasks-0.8.6.gem"
+GEMS="https://rubygems.org/downloads/foreman-tasks-0.9.1.gem"
 GEMS="$GEMS https://rubygems.org/downloads/foreman-tasks-core-0.1.1.gem"
 GEMS="$GEMS https://rubygems.org/downloads/daemons-1.2.4.gem"
 GEMS="$GEMS https://rubygems.org/downloads/parse-cron-0.1.4.gem"
 GEMS="$GEMS https://rubygems.org/downloads/sequel-4.37.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/sinatra-1.4.6.gem"
 GEMS="$GEMS https://rubygems.org/downloads/rack-protection-1.4.0.gem"
-GEMS="$GEMS https://rubygems.org/downloads/dynflow-0.8.17.gem"
+GEMS="$GEMS https://rubygems.org/downloads/dynflow-0.8.23.gem"
 GEMS="$GEMS https://rubygems.org/downloads/algebrick-0.7.3.gem"
 GEMS="$GEMS https://rubygems.org/downloads/apipie-params-0.0.5.gem"
-GEMS="$GEMS https://rubygems.org/downloads/concurrent-ruby-edge-0.2.2.gem"
+GEMS="$GEMS https://rubygems.org/downloads/concurrent-ruby-edge-0.2.3.gem"
```

Comment:
concurrent-ruby-edge 0.2.3 depends sharp on concurrent-ruby 1.0.3, while the Debian Foreman package brings 1.0.5